composer.json

```diff
@@ -33,14 +38,14 @@
         "php": "^8.1",
         "illuminate/auth": "^10.0",
         "illuminate/container": "^10.0",
-        "illuminate/contracts": "^10.0",
-        "jenssegers/mongodb": "^3.9"
+        "illuminate/contracts": "^10.0"
     },
     "require-dev": {
         "monolog/monolog": "^3.2",
         "orchestra/testbench": "^8.0",
         "phpunit/phpunit": "^9.5.10",
-        "squizlabs/php_codesniffer": "^3.7"
+        "squizlabs/php_codesniffer": "^3.7",
+        "mongodb/laravel-mongodb": "v4.0.0-ALPHA2"
```

Comment:
I think you need to move this to the require part ;-)

Reply:
Solved !!